test/src/tests-forwarders.js

```diff
@@ -2250,6 +2250,28 @@ describe('forwarders', function() {
         done();
     });
 
+    it('integration test - after an integration delay is set to false, should fire an event after the event timeout', function(done) {
+        var clock = sinon.useFakeTimers();
```

Comment:
I think we can safely use `let` and `const` in our tests.

```suggestion
        const clock = sinon.useFakeTimers();
```